pkg/k8s/apis/cilium.io/v2/validator/validator.go

```diff
@@ -105,10 +109,90 @@ func (n *NPValidator) ValidateCNP(cnp *unstructured.Unstructured) error {
 	return nil
 }
 
+const (
+	warnEmptyToFromEndpointMessage = "It seems you have a CiliumClusterwideNetworkPolicy " +
+		"with an empty to/from endpoint selector. The behavior of this selector has been " +
+		"changed. The selector now only allows traffic to/from Cilium managed K8s endpoints, " +
+		"instead of acting as a truly empty endpoint selector allowing all traffic. To " +
+		"ensure that the policy behavior does not affect your workloads, consider adding " +
+		"another policy that allows traffic to/from world and cluster entities. For a more " +
+		"detailed discussion on the topic, see GitHub issue #12844"
```

Comment:
When will we want to remove this warning? When 1.10 is out? Once 1.9 is the oldest supported version? It would be nice to add a comment such as `We can remove this once we reach 1.x` so we can grep for it later and don't forget to remove it.

Reply:
This makes sense, I will add a comment regarding this.
One thing I would like to discuss in this context is the backporting scenario for the changes. Do we plan to backport these changes to previous releases? If not we can remove the warnings once the oldest supported version is 1.9

Reply:
Given the upgrade concern, I doubt we can backport this to 1.8/1.7. And unless some users are hitting this, it's probably not worth it anyway.